File: backend/FarmaciaEliseu.Api/Models/Cliente.cs
namespace FarmaciaEliseu.Api.Models;

public class Cliente
{
    public int ClientesId { get; set; }
    public string? Nome { get; set; }
    public string? Endereco { get; set; }
    public string? Rg { get; set; }
    public string? Cpf { get; set; }
    public string? Telefone { get; set; }
    public string? Celular { get; set; }
    public DateTime? DataNascimento { get; set; }
    public int? CodigoFichario { get; set; }
    public DateTime DataCadastro { get; set; }
    public DateTime DataUltimoRegistro { get; set; }
    public bool Deletado { get; set; }
}

File: frontend/src/types/Cliente.ts
export interface Cliente {
  clientesId: number
  nome: string | null
  endereco: string | null
  rg: string | null
  cpf: string | null
  telefone: string | null
  celular: string | null
  dataNascimento: string | null
  codigoFichario: number | null
  dataCadastro: string
  dataUltimoRegistro: string
}

export interface ClienteInput {
  nome: string
  endereco?: string | null
  rg?: string | null
  cpf?: string | null
  telefone?: string | null
  celular?: string | null
  dataNascimento?: string | null
  codigoFichario?: number | null
}
